src/cursor/cur_backup.c

```diff
@@ -267,9 +267,6 @@ __curbackup_close(WT_CURSOR *cursor)
         conn->modified = true;
         WT_TRET(__wt_txn_checkpoint(session, cfg, true));
     }
-    /* Clear the flag on force stop after the completion of the checkpoint. */
-    if (F_ISSET(cb, WT_CURBACKUP_FORCE_STOP))
-        FLD_CLR(conn->log_flags, WT_CONN_LOG_INCR_BACKUP);
```

Comment:
Although it is unfortunately named, this flag and this code need to stay. This code has to do with force stopping block-based incremental backup and whether a system log record needs to be written for the block-based backup IDs.